ckanext/nhm/theme/templates/about/citation.html

```diff
@@ -21,23 +21,43 @@ <h1 class="page-heading">{{ title }}</h1>
-            <p>
-                All of our datasets are assigned a DOI, a persistent digital identifier used to
-                uniquely identify the dataset.  Our DOIs are provided by the
-                <a class="external" target="_blank" href="http://www.bl.uk/aboutus/stratpolprog/digi/datasets/datacitefaq/faqhome.html">
-                    British Library's DataCite service.
-                </a>
-            </p>
-            <p>Please use the DOI in all citations.</p>
-            <h2>Preferred citation format</h2>
-            <p class="about-doi">
-                Scott, B., &amp; Smith , V. (2014): Dataset title data.nhm.ac.uk. https://doi.org/10.5072/1
-            </p>
-        </div>
+    <h2>What to cite</h2>
+    <p>
+        There are a few different ways of citing our data, depending on the material being referenced.
+    </p>
+    <p>Datasets and versioned queries have unique <i>Digital Object Identifiers</i> (DOIs) provided by the <a class="external" target="_blank" href="https://bl.uk/datacite">British Library's DataCite Service</a>. Please use these if you cite these materials in your work.</p>
+
+    <h3>1. Datasets</h3>
+    <p>
+        A single, complete dataset: e.g. the DOI <a href="https://doi.org/10.5519/hay2d5yo">10.5519/hay2d5yo</a> references the <a href="/dataset/join-the-dots-collection-level-descriptions">Join the Dots</a> dataset (including all resources therein).
+    </p>
+    <p>
+        Each dataset overview page will have a 'Cite this as' section with an APA-style reference you can copy, or you can download the BibTeX reference from DataCite using the "BibTeX" button.
+    </p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Dataset name</i></u> [Data set]. Natural History Museum. https://doi.org/<u>doi</u>
     </div>
+
+    <h3>2. Resources</h3>
+    <p>We don't currently provide separate DOIs for resources, so if you cite a single resource use the standard URL. As with datasets, each resource overview page has an APA-style example reference that can be copied.</p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Resource name</i></u> (from <u><i>Dataset name</i></u>) [Data set]. Natural History Museum. https://data.nhm.ac.uk/dataset/<u>dataset-id</u>/resource/<u>resource-id</u>
+    </div>
+
+    <h3>3. Records</h3>
+    <p>Records are versioned, so you can either cite the most up-to-date version of the record or the version as it was when you initially retrieved it. Both URLs can be found in the 'Cite this as' section at the bottom of a record page.</p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Record name</i></u> (from <u><i>Dataset name</i></u>, <u><i>Resource name</i></u>) [Data set record]. Natural History Museum. https://data.nhm.ac.uk/object/<u>record-id</u>/<u>version</u>
+    </div>
+
+    <h3>4. Searches</h3>
```

Comment:
Do we want to mention the old search interface here too?

Reply:
Would these just be cited like a website?

Reply:
they' be the same as the new search interface cause you also get a DOI through the old search interface driven download mechanism too (i.e. through the ckanpackager). It's just a different way to get to essentially the same result, a DOI for your download. Here's an example I just made by downloading through the old search interface: https://data.nhm.ac.uk/doi/10.5519/qd.ocr0c1vw.